data/json/items/gunmod/underbarrel.json

```diff
@@ -242,22 +242,19 @@
     "name": "integrated bayonet",
     "description": "This is the bayonet integrated in the firearm.",
     "weight": 1,
-    "volume": 0,
+    "volume": 1,
     "price": 4000,
     "to_hit": -1,
     "cutting": 10,
     "material": [ "steel" ],
     "symbol": ":",
     "color": "light_gray",
-    "flags": [ "STAB" ],
-    "gunmod_data": {
-      "location": "underbarrel",
-      "mod_targets": [ "shotgun", "rifle" ],
-      "dispersion_modifier": 10,
-      "mode_modifier": [ [ "REACH", "bayonet", 2, [ "MELEE" ] ] ],
-      "min_skills": [ [ "weapon", 2 ], [ "melee", 1 ] ],
-      "flags": [ "SLOW_WIELD", "IRREMOVABLE" ]
-    }
+    "location": "underbarrel",
+    "mod_targets": [ "shotgun", "rifle" ],
+    "dispersion_modifier": 10,
+    "mode_modifier": [ [ "REACH", "bayonet", 2, [ "MELEE" ] ] ],
+    "min_skills": [ [ "weapon", 2 ], [ "melee", 1 ] ],
+    "flags": [ "SLOW_WIELD", "IRREMOVABLE", "STAB" ]
```

Comment:
When I added a gunmod with min_skills, the game wouldn't allow its use until the character had the skills specified. You might want to remove those until later